### DISCUSSION TITLE
How do i verify the "authjs.session-token" in express?

### DISCUSSION THREAD

**Discussion by shivamrun** [nextauthjs/next-auth#10955] [Help] [answered]
I have implemented Google OAuth 2.0 in nextjs using next-auth
and all my applications backend logic is written on express server
but am not getting that how to verify the requests coming on the server

below are the cookies i sended from next-app to express using axios + { withCredentials: true }
![Screenshot 2024-05-19 005501](https://github.com/nextauthjs/next-auth/assets/108524973/7772e49c-9191-4620-bf07-d76474153934)

how do i verify a user on express js?
even the "authjs.session-token" is not decoded/decrypted! :(

am stuck on this step from 2 weeks
trying googling, gpting, binging in youtube but not finding any suitable solution

**Comment by Goldfish7718** [comment]
Hi, did you find the solution?

**Reply by shivamrun** [reply]
nahh bro

but in new version of auth.js
they've provided support for express, but the documentation for this is partially completed.

**Reply by hillac** [reply, marked as answer]
I haven't tried it but this may work: https://github.com/nextauthjs/next-auth/discussions/10844#discussioncomment-10170743

**Reply by Goldfish7718** [reply]
I will try this and let you know if this works

**Reply by Goldfish7718** [reply]
> I haven't tried it but this may work: https://github.com/nextauthjs/next-auth/discussions/10844#discussioncomment-10170743

Hey it worked!
